tools/topology/m4/tdfb_controls.m4

```diff
@@ -0,0 +1,33 @@
+# TDFB Bytes control with max value of 255
+C_CONTROLBYTES(DEF_TDFB_BYTES, PIPELINE_ID,
+	CONTROLBYTES_OPS(bytes,
+		258 binds the mixer control to bytes get/put handlers,
+		258, 258),
+	CONTROLBYTES_EXTOPS(258 binds the mixer control to bytes get/put handlers,
+		258, 258),
+	, , ,
+	CONTROLBYTES_MAX(, 4096),
+	,
+	DEF_TDFB_PRIV)
+
+# TDFB enum list
+CONTROLENUM_LIST(DEF_TDFB_ONOFF, LIST(`	', `"beam on"', `"beam off"'))
```

Comment:
Here are the on/off control names. Please comment if you have in mind something better.

Reply:
Same, the names can be improved when the kernel supports the controls.